### PR TITLE
Evaka create citizen subnavigation

### PR DESCRIPTION
**Pull request by mapehe**
#### Summary

Group some navbar tabs under a single `Applying`-tab with a sub-navbar.

image

### CODE REVIEW

File: frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx [hunk 1]
```diff
@@ -71,7 +71,8 @@ const sv: Translations = {
       decisions: 'Beslut',
       income: 'Inkomst',
       messages: 'Meddelanden',
-      calendar: 'Kalender'
+      calendar: 'Kalender',
+      applying: 'Ansöker'
```
Comment (Joosakur) [inline frontend/src/lib-customizations/espoo/citizen/assets/i18n/sv.tsx:75]: has this been checked from somewhere?

File: frontend/src/citizen-frontend/map/MapView.tsx [hunk 2]
```diff
@@ -96,12 +115,14 @@ export default React.memo(function MapView() {
   useTitle(t, t.map.title)
 
   return (
-    <FullScreen data-qa="map-view">
+    <MapFullscreenContainer loggedIn={loggedIn}>
```
Comment (Joosakur) [inline frontend/src/citizen-frontend/map/MapView.tsx:118]: Screenshot 2021-09-28 at 12 05 10
Something wrong with the map. Sometimes that gray area goes away when moving the map but reappears on refresh. No issue when logged out.

**Review by Joosakur — CHANGES_REQUESTED** [review]

**Comment by Joosakur** [comment]
in the spec there's a gap between the top nav and secondary tabs
Screenshot 2021-09-28 at 12 11 38

**Comment by mapehe** [comment]
Thanks for the feedback @Joosakur, suggested some fixes in 0703968.

**Review by Joosakur — APPROVED** [review]

**Pull request merged**